plugins/rplanners/rplanners.h

```diff
@@ -126,7 +126,7 @@ class SpatialTreeBase
 
     /// extends toward pNewConfig
     /// \return true if extension reached pNewConfig
-    virtual ExtendType Extend(const std::vector<dReal>& pTargetConfig, NodeBasePtr& lastnode, bool bOneStep=false) = 0;
+    virtual ExtendType Extend(const std::vector<dReal>& pTargetConfig, NodeBasePtr& lastnode, bool bOneStep=false, bool saveStatusData=false) = 0;
```

Comment:
It's weird that you call it `saveStatusData` and have a enum named `PO_StatusDetail` and what it does is add `CFO_FillCollisionReport` to options to `CheckPathAllConstraints`. None of these 3 names are related to each other.